### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by mikhailivich** [entropyxyz/community#130] [Get Test Funds] [answered]
### Address

5Fes3ciLtT8nRF3RwYsMYRCBzmNAxbkeEoxsmkRbVWoWVbET

**Comment by johnnymatthews** [comment, marked as answer]
Sorry for the delay. Sent in block [613,520](https://polkadot.js.org/apps/#/explorer/query/0x5949c70402235f2781dc980a96b63a639791ffc9f73f0e9b73774a6bad347bca)
